components/reusable-components/blog-card/BlogCard.tsx

```diff
@@ -14,9 +14,9 @@ export interface BlogCardProps {
   pageTitle: string;
 }
 
-const BlogCard = ({ title, body, pageTitle }: BlogCardProps) => {
+const BlogCard = ({ id, title, body, pageTitle }: BlogCardProps) => {
   return (
-    <Link href={`/blog/${title}`} className={style.blogCardContainer}>
+    <Link href={`/blog/${id}`} className={style.blogCardContainer}>
```

Comment:
The URL of the individual blog post should contain the title:
https://learnhub.mk/blog/blog-title